testing/scenario_app/ios/Scenarios/ScenariosTests/FlutterEngineTest.m

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <XCTest/XCTest.h>
+#import "AppDelegate.h"
+
+@interface FlutterEngineTest : XCTestCase
+@end
+
+@implementation FlutterEngineTest
+XCTestExpectation* _isolateIdSetExpectation;
```

Comment:
I think `keyValueObservingExpectationForObject:::` should work here so you don't need to set up your own KVO observation.